Document that unique property constraints can be composite (#1176)
neo4j/neo4j-documentation: cypher/cypher-docs/src/docs/dev/glossary.asciidoc

|<<query-call-subquery, CALL {...}>>        | Reading/Writing   | Evaluates a subquery, typically used for post-union processing or aggregations.
|<<query-create, CREATE>>                    | Writing     |  Create nodes and relationships.
|<<administration-constraints-syntax, CREATE CONSTRAINT [existence] [IF NOT EXISTS] ON (n:Label) ASSERT n.property IS NOT NULL>>  | Schema   | Create a constraint ensuring that all nodes with a particular label have a certain property.

> |<<administration-constraints-syntax, CREATE CONSTRAINT [node_key] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS NODE KEY [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint ensuring all nodes with a particular label have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.

Reviewer: ```suggestion
|<<administration-constraints-syntax, CREATE CONSTRAINT [node_key] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS NODE KEY [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint that ensures that all nodes with a particular label have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
```

> |<<administration-constraints-syntax, CREATE CONSTRAINT [node_key] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS NODE KEY [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint that ensures all nodes with a particular label have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.

Reviewer: This sentence could keep the extra _that_ : "...that ensures that all nodes.."

Author: ```suggestion
|<<administration-constraints-syntax, CREATE CONSTRAINT [node_key] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS NODE KEY [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint that ensures that all nodes with a particular label have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
```

|<<administration-constraints-syntax, CREATE CONSTRAINT [existence] [IF NOT EXISTS] ON ()-"["r:REL_TYPE"]"-() ASSERT r.property IS NOT NULL>>  | Schema   | Create a constraint ensuring that all relationships with a particular type have a certain property.

> |<<administration-constraints-syntax, CREATE CONSTRAINT [uniqueness] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS UNIQUE [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint ensuring the uniqueness of the combination of node label and property values for a particular property key combination across all nodes.

Reviewer 2: Looks fine, but again these links are broken both here and in the real docs

Reviewer: ```suggestion
|<<administration-constraints-syntax, CREATE CONSTRAINT [uniqueness] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS UNIQUE [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint that ensures that the uniqueness of the combination of node label and property values for a particular property key combination across all nodes.
```

> |<<administration-constraints-syntax, CREATE CONSTRAINT [uniqueness] [IF NOT EXISTS] ON (n:Label) ASSERT (n.prop1[, ..., n.propN]) IS UNIQUE [OPTIONS {optionKey: optionValue[, ...]}]>>  |  Schema | Create a constraint that ensures that the uniqueness of the combination of node label and property values for a particular property key combination across all nodes.

Reviewer: Ah, my butterfingers! One too many _that_. 
...Create a constraint that ensures the uniqueness...
